arangod/Cache/Manager.cpp

```diff
@@ -28,12 +28,12 @@
 #include <set>
 #include <thread>
 #include <utility>
+#include "Basics/cpu-relax.h"
```

Comment:
why is this included moved to the top here and in other files?

Reply:
Check last commit :( I tried to replace

https://github.com/arangodb/arangodb/pull/18448/commits/1365553b076454621d0e225eb6a7e99bb53e6889

Reply:
It wasn't intentional as you can see

Reply:
Fixed